_troubleshoot/index.md

```diff
@@ -116,3 +116,12 @@ The security plugin blocks the update by script operation (`POST <index>/_update
 ## Illegal reflective access operation in logs
 
 This is a known issue with Performance Analyzer that shouldn't affect functionality.
+
+
+## bootstrap checks failed: max virtual memory areas vm.max_map_count is too low
+
+If you're using a production workload, be sure to set the [Linux setting](https://www.kernel.org/doc/Documentation/sysctl/vm.txt) ```vm.max_map_count``` to 262144
+
+To increase memory and make a change permanent
+edit /etc/sysctl.conf and set ```vm.max_map_count``` to 262144
+you can check it using ```sysctl vm.max_map_count```
```

Comment:
Since we have the previous sentence, this section isn't necessary. Also, to avoid repeating information, I would link to the [important settings](https://opensearch.org/docs/latest/opensearch/install/important-settings/) page as in my previous suggestion